net-im/qtox/qtox-1.7.0-r1.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit qmake-utils toolchain-funcs
+
+DESCRIPTION="Most feature-rich GUI for net-libs/tox using Qt5"
+HOMEPAGE="https://github.com/qTox/qTox"
+SRC_URI="https://github.com/qTox/qTox/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk X"
+
+# needed, since tarball provided by github extracts to `qTox`
+S="${WORKDIR}/qTox-${PV}"
+
+RDEPEND="
+	dev-db/sqlcipher
+	dev-libs/libsodium
+	dev-qt/qtconcurrent:5
+	dev-qt/qtcore:5=
```

Comment:
I've been told by @kensington that there's usually no need for this and in fact the metadata.xml of qtcore also says the same thing: `Must only be used by packages that are known to use private parts of the Qt API.` If there anything to justify this?